### PR TITLE
LPS-127003 Fail to review changes after upgrading DDM within Publications from 7.3

### PR DESCRIPTION
**Pull request by stsquared99**

### CODE REVIEW

File: modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java [hunk 1]
```diff
@@ -891,35 +893,55 @@ else if (ctEntry.getChangeType() ==
 
 				T model = null;
 
-				if ((ctCollectionId == _ctCollection.getCtCollectionId()) &&
-					(ctSQLMode == CTSQLModeThreadLocal.CTSQLMode.DEFAULT)) {
+				String logMessage = StringBundler.concat(
```
Comment (Preston-Crary) [inline modules/apps/change-tracking/change-tracking-web/src/main/java/com/liferay/change/tracking/web/internal/display/context/ViewChangesDisplayContext.java:896]: Let's make this message lazy. Maybe make a method `_getMissingModelMessage(...)`.

**Comment by stsquared99** [comment]
@Preston-Crary updated

**Comment by Preston-Crary** [comment]
ci:forward

**Comment by Preston-Crary** [comment]
ci:forward

**Pull request closed without merging**